geomstats/geometry/euclidean.py

```diff
@@ -218,3 +195,195 @@ def parallel_transport(
+        end_point : array-like, shape=[..., dim], optional
+            Point on the manifold, end point of the geodesic. If None,
+            an initial tangent vector must be given.
+        initial_tangent_vec : array-like, shape=[..., dim],
+            Tangent vector at base point, the initial speed of the geodesics.
+            Optional, default: None.
+            If None, an end point must be given and a logarithm is computed.
+
+        Returns
+        -------
+        path : callable
+            Time parameterized geodesic curve. If a batch of initial
+            conditions is passed, the output array's first dimension
+            represents the different initial conditions, and the second
+            corresponds to time.
+        """
+        if end_point is None and initial_tangent_vec is None:
+            raise ValueError(
+                "Specify an end point or an initial tangent "
+                "vector to define the geodesic."
+            )
+        if end_point is not None:
+            if initial_tangent_vec is not None:
+                raise ValueError(
+                    "Cannot specify both an end point and an initial tangent vector."
+                )
+
+            initial_tangent_vec = self.log(end_point, initial_point)
+
+        is_batch = check_is_batch(
+            self._space.point_ndim, initial_point, initial_tangent_vec
+        )
+        if is_batch:
+            initial_point = gs.expand_dims(
+                initial_point, axis=-(self._space.point_ndim + 1)
+            )
+
+        ijk = "ijk"[: self._space.point_ndim]
+
+        def path(t):
+            """Generate parameterized function for geodesic curve.
+
+            Parameters
+            ----------
+            t : array-like, shape=[n_points,]
+                Times at which to compute points of the geodesics.
+            """
+            t = gs.array(t)
+            t = gs.cast(t, initial_tangent_vec.dtype)
+            t = gs.to_ndarray(t, to_ndim=1)
+            tangent_vecs = gs.einsum(f"n,...{ijk}->...n{ijk}", t, initial_tangent_vec)
+            return initial_point + tangent_vecs
+
+        return path
+
+    def injectivity_radius(self, base_point):
+        """Compute the radius of the injectivity domain.
+
+        This is is the supremum of radii r for which the exponential map is a
+        diffeomorphism from the open ball of radius r centered at the base
+        point onto its image.
+
+        Parameters
+        ----------
+        base_point : array-like, shape=[..., {dim, [n, m]}]
+            Point on the manifold.
+
+        Returns
+        -------
+        radius : array-like, shape=[...,]
+            Injectivity radius.
+        """
+        radius = gs.array(math.inf)
+        return repeat_out(self._space.point_ndim, radius, base_point)
+
+
+class StandardEuclideanMetric(EuclideanMetric):
+    """Class for the standard Euclidean metric."""
```

Comment:
Explain what is "standard", explain the difference with the EuclideanMetric, eg in a See Also section in the docstring.